Update day name

davidekunno93 opened the pull request:
### Update Day Name

**Summary of changes**
1 route added in /app/places/routes.py

`update_day_name`: This route expects a the database id of the day to be updated in the trail end of the URL i.e. .../update-day-name/{day_id}. The route also expects a package with the new day name to be served to the route as a value of a key named `dayName`. This route has been tested on my local machine and worked successfully. The use case for this route is in the itinerary page, when users add or update the subheading of any trip day this route is contacted on the backend to save those changes.

kaitlynnchan reviewed (commented):
can you give an example of the before and after change would look like? im a bit confused what day_name is supposed to be.

davidekunno93 commented:
### Summary of changes

- Replaced .get query method with .filter_by instead
- Added data change validation check to confirm the Day object's day_name property has been updated before responding with status 200

Route tested on local machine and is working. 

### Limitations/Issues
- After updating the Day object with new day_name, the database seems to reorganize the order of the Day objects so that the next time the trip is viewed the Days aren't always in correct order (the Day that was last updated is displayed at the bottom). The places/get_trip route should organize the Day objects in chronological order before returning the data, this would resolve this issue.

kaitlynnchan reviewed (commented):
Just one more thing, could you add sample request/response to the readme file, thanks!
